nilearn/plotting/html_stat_map.py

```diff
@@ -1,157 +1,488 @@
 """
-Visualizing 3D stat maps in a Papaya viewer
+Visualizing 3D stat maps in a Brainsprite viewer
 """
+import warnings
 import os
-import tempfile
-import base64
 import json
-import warnings
+from io import BytesIO
+try:
```

Comment:
You would add a `encodebytes` in `_utils.compat`, that simply points to the import above? That seems easy enough to do if you think this will be useful in other parts of nilearn.

Reply:
I have added a _utils.compat._encodebytes method, which I re-use in html_stat_map.